### ISSUE TITLE
Move alert strings to separate language file

### ISSUE THREAD

**Issue by titanomachy** [icook/yota#81]
The text for validation alerts is now contained in the validators classes. Replacing the strings with tags allows for multi language alerts and customization of the alert messages.

**Comment by icook** [comment]
This is a good idea, but will probably be a bit before I implement it. There are some bigger features that are my focus at the moment. If you want to implement it and make a pull request you're welcome to.

**Comment by titanomachy** [comment]
Yeah sure, I will take a look at it. The bigger picture is more important indeed.
